AsyncDisplayKit/Layout/ASLayoutElementExtensibility.h

```diff
@@ -30,5 +34,80 @@
 - (void)setLayoutOptionExtensionEdgeInsets:(UIEdgeInsets)value atIndex:(int)idx;
 - (UIEdgeInsets)layoutOptionExtensionEdgeInsetsAtIndex:(int)idx;
 
-
 @end
+
+#pragma mark - Dynamic Properties
+
+/**
+ * Unbox NSNumber based on the type
+ */
+#define ASDK_UNBOX_NUMBER(NUMBER, PROPERTY_TYPE) \
```

Comment:
Is there any concern that this doesn't support other values that NSNumber does? I.e. long, longlong, char, etc?